slug.go

```diff
@@ -75,19 +75,27 @@ func packWalkFn(root, src, dst string, tarW *tar.Writer, meta *Meta, dereference
 			return nil
 		}
 
-		// Ignore the .terraform directory itself.
-		if info.IsDir() && info.Name() == ".terraform" {
+		// Ignore the .terraform and terraform.d directories.
+		if info.IsDir() && info.Name() == ".terraform" || info.Name() == "terraform.d" {
 			return nil
```

Comment:
Is it possible to `return filepath.Skipdir` earlier and skip some of the logic below, or am I missing something?

Reply:
We cannot return `filepath.Skipdir` here because we want to include `.terraform/modules`

Reply:
🤦‍♀ yes thank you, that makes perfect sense, sorry!

Reply:
No problem at all! 👍